itsnameissteven/Refactor-Tractor-Fitlit
# Feature/user repo refactor

## Pull Request

### Description

* Is this a feature or a fix?
This is a fix
* Where should the reviewer start?
Review the User-test then head over to user-repository test. Finally review the UserRepo class
* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
No functionality changed
* What outside features or research did you use to implement this fix?
n/a
* How should this be tested?
Run npm test. Also check out the app to make sure everything displays as expected.

### Relevant Screen Shots
***
#### Please review considerations below:
